### ISSUE TITLE
Madeline fails on one single individual pedigree

### ISSUE THREAD

**Issue by ingkebil** [CGHQ/mip2scout#1]
Which might be fine. But it does halt the further execution of the script and this not creating a configuration file for the upload step.

**Comment by ingkebil** [comment]
```
$ for SAMPLE in 15078-miptest; do echo $SA
MPLE; mip2scout -c /home/scout/components/mip2scout/config/main.yaml preprocess -r /home/scout/components/mip2scout /mnt/hds/proj/bioinfo/develop/cust000/${SAMPLE}/exomes/${SAMPLE}; 
  done
15078-miptest
[2015-12-07 09:38:02,550] INFO    : mip2scout.cli.root       : pre-processing: cust000-15078-miptest
[2015-12-07 09:38:02,560] INFO    : mip2scout.cli.root       : load coverage for ADM1101A6-miptest
[2015-12-07 09:38:02,677] ERROR   : mip2scout.coverage       : sample already loaded, rolling back
[2015-12-07 09:38:04,165] INFO    : mip2scout.cli.root       : generate madeline SVG file
Traceback (most recent call last):
  File "/home/scout/miniconda/envs/scout12/bin/mip2scout", line 9, in <module>
    load_entry_point('mip2scout', 'console_scripts', 'mip2scout')()
  File "/home/scout/miniconda/envs/scout12/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/home/scout/miniconda/envs/scout12/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/home/scout/miniconda/envs/scout12/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/scout/miniconda/envs/scout12/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/scout/miniconda/envs/scout12/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/home/scout/miniconda/envs/scout12/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/scout/components/mip2scout/src/mip2scout/mip2scout/cli/root.py", line 127, in preprocess
    madeline_file = run_madeline(mip_case, case_root, madeline_exe)
  File "/home/scout/components/mip2scout/src/mip2scout/mip2scout/cli/root.py", line 30, in run_madeline
    svg_content = madeline.run(mip_case.ped, exe=madeline_exe)
  File "/home/scout/components/mip2scout/src/mip2scout/mip2scout/madeline.py", line 28, in run
    raise SinglePedigreeError("can't generate pedigree with single"
mip2scout.exc.SinglePedigreeError: can't generate pedigree with singleindividual
```

**Comment by moonso** [comment]
@ingkebil you could just catch that error and continue, set madeline path to nothing.

**Comment by ingkebil** [comment]
Fixed 8bf68db62
